src/Microsoft.AspNetCore.SignalR.Protocols.Json/JsonHubProtocolOptions.cs

```diff
@@ -6,8 +6,14 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Options used to configure a <see cref="JsonHubProtocol"/> instance.
+    /// </summary>
     public class JsonHubProtocolOptions
     {
+        /// <summary>
+        /// Gets or sets the settings used to serialize payloads.
```

Comment:
~~payloads~~ arguments and return values.

(I know the property says "payload" but that's because "ArgumentAndReturnValueSerializerSettings" is doofy. Let's explain what "payload" means in the docs)